ajax/libs/imgcache.js/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "imgcache.js",
+  "version": "2.0.0",
+  "description": "JS library based on the File API to cache images for offline recovery (target: cordova/phonegap & chrome)",
+  "filename": "imgcache.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chrisben/imgcache.js.git"
+  },
+  "keywords": [
+    "image",
+    "cache",
+    "html5",
+    "file",
+    "filesystem",
+    "offline",
+    "cordova",
+    "phonegap",
+    "imgcache"
+  ],
+  "author": "Christophe BENOIT",
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/chrisben/imgcache.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/?(q)imgcache*.+(css|map|js)"
+        ]
```

Comment:
@sashberd This is a discussion.
Would you please remove this part and manually add old versions?
Because in my view, you can see the file structure with `imgcache.js` file is different from v2.0.0 and the versions before v1.2.0.
One is `ajax/libs/imgcache.js/2.0.0/imgcache.js`, another is `ajax/libs/imgcache.js/1.2.0/js/imgcache.js`.
It not very intuitional to developers, because the different file structure.
The best way is make v2.0.0 and old versions with the same file structure. Also, the file structure of old versions should follow the latest version's file structure.

However, we don't use `js` as the basePath in auto-update config, so the good way is manually ass old versions.
What do you think?
Thank you.